src/bin/common.rs

```diff
@@ -322,6 +323,12 @@ pub fn parse_cli() -> Result<CliOptions, CliError> {
         .long("verbose")
         .short("v")
     )
+    .arg(
+      Arg::with_name("QUIET")
+        .help("Do not output any status message")
+        .long("quiet")
+        .short("q")
```

Comment:
What is the desired behavior if both quiet and verbose are specified? Should we assume one, or display an error?

Reply:
Quiet beats verbose in my implementation.